src/styles/blocks/contacts.scss

```diff
@@ -0,0 +1,124 @@
+
+.contact__container {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 120px;
+
+  @include OnDesktop-744 {
+    display: grid;
+    justify-content: normal;
+    margin-bottom: 180px;
+  }
+}
+
+.contact__info {
+  &-title {
+    display: block;
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 19.6px;
+
+    margin-bottom: 8px;
+  }
+
+  &-link {
+    display: block;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 19.6px;
+    color: #fff;
+    margin-bottom: 24px;
+
+    text-decoration: none;
+  }
+}
+
+.contact__form {
+  &-label {
+    display: block;
+  }
+
+  &-name,
+  &-email {
+    height: 48px;
+    width: 288px;
+
+    background: #f8f8fa;
+    border: none;
+    border-radius: 40px;
+    margin-bottom: 16px;
+    box-sizing: border-box;
+    padding: 16px;
+
+    &:hover {
+      font-weight: 400;
+      font-size: 14px;
+      line-height: 20px;
+      color: #292929;
+
+      box-sizing: border-box;
+      border: 1px solid #d5d7de;
+    }
+  }
+
+  &-name::placeholder,
+  &-email::placeholder,
+  &-message::placeholder {
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 20px;
+    color: #7e7e83;
+  }
+
+  &-name[type="text"],
+  &-email[type="email"],
+  &-message[name="message"] {
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 20px;
+    color: $secondColor;
+  }
+
+  &-message {
+    width: 288px;
+    height: 148px;
+
+    background: #f8f8fa;
+    border: none;
+    box-sizing: border-box;
+    border-radius: 16px;
+    margin-bottom: 32px;
+    padding: 14px 16px;
+  }
```

Comment:
placeholder doesn't inherit font-family
you should add: font-family: inherit;
![image](https://user-images.githubusercontent.com/87998640/223983021-092c0377-9d54-4e30-819a-86614365a462.png)